specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/disasterRecoveryConfigurations.json

```diff
@@ -0,0 +1,405 @@
+    },
+    "ProxyResource":{
+      "description":"ARM proxy resource.",
+      "allOf":[
+        {
+          "$ref":"#/definitions/Resource"
+        }
+      ]
+    },
+    "DisasterRecoveryConfigurationProperties": {
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The status of the disaster recovery configuration."
+        },
+        "autoFailover": {
+          "type": "string",
+          "description": "Whether or not failover can be done automatically.",
+          "enum": ["Off", "On"],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "DisasterRecoveryConfigurationAutoFailover"
+          }
+        },
+        "failoverPolicy": {
+          "type": "string",
+          "description": "How aggressive the automatic failover should be.",
+          "enum": ["Off", "On"],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "DisasterRecoveryConfigurationFailoverPolicy"
+          }
+        },
+        "partnerServerId": {
+          "type": "string",
+          "description": "Id of the partner server."
+        },
+        "logicalServerName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Logical name of the server."
+        },
+        "partnerLogicalServerName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Logical name of the partner server."
+        },
+        "role": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The role of the current server in the disaster recovery configuration.",
+          "enum": ["Primary", "Secondary"],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "DisasterRecoveryConfigurationRole"
+          }
+        }
+      },
+      "required": ["autoFailover", "failoverPolicy", "partnerServerId"],
+      "description": "Represents the properties of a disaster recovery configuration."
+    },
+    "DisasterRecoveryConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DisasterRecoveryConfigurationProperties",
+          "description": "The properties representing the resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Type of resource this is."         
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the server that contains this disaster recovery configuration.",
+          "readOnly": true
+        }        
+      },
+      "required": [
+        "properties"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Represents a disaster recovery configuration."
+    },
+    "DisasterRecoveryConfigurationListResult": {
```

Comment:
Are we missing `DisasterRecoveryConfigurations_List` operation? because i don't see this model being used anywhere.

Reply:
I did not have this earlier. It is included now.